Extensions/retags.js

```diff
@@ -91,6 +91,7 @@ XKit.extensions.retags = {
 
 	tag: function(elements) {
 		$(elements).each(function() {
+			$(this).addClass("retags_has_processed");
 			var $element = $(this),	retagClass, $target, url, host, id;
```

Comment:
we can just use $element instead of `$(this)`

Reply:
$element is only used at the very end (when retags are added), which is the entire problem. because the tag function gets called 3 times, it needs to add the class first before the subsequent (and almost instant) 2 times can find anything to work on